test/benchmarks/mongoBench/suites/parallelBench.js

```diff
@@ -0,0 +1,206 @@
+const { createReadStream, createWriteStream } = require('fs');
+const { rm, mkdir, readdir } = require('fs/promises');
+const { resolve } = require('path');
+const { Readable, once } = require('stream');
+const readline = require('readline');
+const {
+  makeClient,
+  disconnectClient,
+  dropDb,
+  initBucket,
+  dropBucket,
+  initCollection,
+  initDb,
+  connectClient,
+  createCollection
+} = require('../../driverBench/common');
+const { pipeline } = require('stream/promises');
+const { EJSON } = require('bson');
+
+const benchmarkFileDirectory = resolve(__dirname, '..', '..', 'driverBench', 'spec', 'parallel');
+
+async function initTemporaryDirectory() {
+  const temporaryDirectory = resolve(benchmarkFileDirectory, 'downloads');
+  await rm(temporaryDirectory, { recursive: true, force: true }).catch(() => {
+    // ignore errors, specifically directory not found
```

Comment:
Specifying `force:true` will make it silent for non-existent directories so do we need the catch? it could silence something important

Reply:
What important errors do you foresee us suppressing?  This isn't part of the benchmark, it's just setup code so any errors aren't directly relevant to the benchmark.  I can change it if you want me to but otherwise I'll leave it